database/Group5Database.py

```diff
@@ -49,14 +49,19 @@ def get_graph(self):
             'messages': messages
         }
 
-    def get_messages(self):
+    def get_messages(self, sentiment_type="NEUTRAL"):
         result = self.get_graph()
-        return result['messages']
+        if sentiment_type == 'POSITIVE':
+            return list(filter(lambda x: x['sentiment'] >= 0, result['messages']))
+        elif sentiment_type == 'NEGATIVE':
+            return list(filter(lambda x: x['sentiment'] <= 0, result['messages']))
+        else:
```

Comment:
...und hier dann die Bedingung in die Cypher Query einarbeiten, wie bei Gruppe 4. Dann werden die nicht benötigen Daten garnicht erst aus der Datenbank geladen.